<launch>

	<node name="copter_control" pkg="copter_control" type="copter_control" output="screen">

		<!-- Pilot Command rate -->
		<param name="rate" value="20.0"/>

		<param name="enable_xy_stab" value="false"/>
		<param name="rc_enable_xy_ch" value="5"/>
		
		<!-- max pithc\roll\z errors -->
		<param name="max_pitch" value="3.0"/>
		<param name="max_roll" value="3.0"/> 
		<param name="max_xy_vel" value="0.8"/>
		<param name="max_z_vel" value="0.25"/> 
		<param name="min_z_vel" value="-0.25"/> 

		<!-- XY command (pithc\roll) PID gains -->
		<param name="xy_p_gain" value="4.0"/>
		<param name="xy_i_gain" value="1.0"/> 
		<param name="xy_d_gain" value="0.05"/> 
		<param name="xy_i_max" value="1.0"/>
		<param name="xy_i_min" value="-1.0"/> 

		<!-- XY velocity goal PID gains -->
		<param name="xy_vel_goal_p_gain" value="1.0"/> 
		<param name="xy_vel_goal_i_gain" value="0.06"/>  
		<param name="xy_vel_goal_d_gain" value="1.2"/> 
		<param name="xy_vel_goal_i_max" value="0.1"/>
		<param name="xy_vel_goal_i_min" value="-0.1"/>

		<!-- Z command PID gains -->
		<param name="z_p_gain" value="1.2"/>  <!--1.2-->
		<param name="z_i_gain" value="0.13"/> <!-- 0.13-->
		<param name="z_d_gain" value="0.0"/> 
		<param name="z_i_max" value="0.2"/>
		<param name="z_i_min" value="-0.2"/> 

	</node>

	<node name="copter_mavros" pkg="copter_mavros" type="copter_mavros">

		<param name="throttle_z_vel" value="1000.0"/>  <!--750  
								- maximum vertical speed 0.5 m/s PILOT_VELZ_MAX = 50
								- size of the deadband throttle THR_DZ = 50 -->
		<param name="throttle_center" value="1500.0"/>
		<param name="pitch_deg" value="33.333"/> 
		<param name="pitch_center" value="1500.0"/>
		<param name="roll_deg" value="33.333"/>  
		<param name="roll_center" value="1500.0"/>

		<param name="rc_control_ch" value="7"/>
	</node>


</launch>
